src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -14,107 +15,94 @@ import dataLeft from './demo/data--align-left';
 import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
-const PrepareBanner = data => {
+const prepareBanner = data => {
+  if (data.title) {
+    data.title = text('title', data.title, buttonLabels.required);
+  }
+
+  if (data.link.link.label) {
+    data.link.link.label = text(
+      'link.Link.label',
+      data.link.link.label,
+      buttonLabels.required
+    );
+  }
+  if (data.description) {
+    data.description = text(
+      'description',
+      data.description,
+      buttonLabels.required
+    );
+  }
+
+  if (data.centered) {
+    data.centered = boolean('centered', data.centered, buttonLabels.optional);
+  }
+
+  if (data.image) {
+    data.image = text('image', data.image, buttonLabels.optional);
+  }
   data.link.icon.path = defaultSprite; // eslint-disable-line no-param-reassign
```

Comment:
Ah, so this should turn into a knob select with the only option being the current path for the icons file.
But you have the exclusion for this eslint check at the top of the file, you don't really need to exclude the check in this line in particular, @Joosthe